packages/components/src/components/form/SelectBar/index.tsx

```diff
@@ -134,7 +124,13 @@ const SelectBar: <V extends ValueTypes>(props: SelectBarProps<V>) => JSX.Element
 
     return (
         <Wrapper className={className} isInLine={isInLine} {...rest}>
-            {label && <Label isInLine={isInLine}>{label}</Label>}
+            {/* {label && <Label isInLine={isInLine}>{label}</Label>} */}
```

Comment:
forgotten comment?

Reply:
Pushed while in the middle of something.

Reply:
still here?